backend.dev.Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM node:alpine3.17 AS builder
```

Comment:
I think we should specify Node version we want this to run on

Reply:
Yeah I agree... `node:18-alpine3.17` ?

Reply:
node-v20.8.1

Reply:
@chrlschwb it would be nice but v20 hasn't been released as a LTS yet it should happen soon but rn v18 is still the most recent LTS